### PR TITLE
Documentation for Windows Setup

### PR DESCRIPTION
**Pull request by Jdew192837**
Added a new section for "Windows Setup" in the install documentation.

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -275,6 +276,84 @@ librehealthehr/sites/default/letter_templates
...
+##  Windows Setup
+
+#### Setup
+To run LibreEHR on Windows, [XAMPP](https://sourceforge.net/projects/xampp/files/XAMPP%20Windows/5.6.30/) is needed with a compatible version
+
+**Note:** 
+
+1. Must have php [7.0](https://sourceforge.net/projects/xampp/files/XAMPP%20Windows/_) or [5.6](https://sourceforge.net/projects/xampp/files/XAMPP%20Windows/) ([5.6.30](https://sourceforge.net/projects/xampp/files/XAMPP%20Windows/5.6.30/) recommended)
+
+2. php 7.1x isn't supported
```
Comment (teryhill) [inline INSTALL.md]: Make the currently un supported. It will be when I can get you to fix it after GSI

File: INSTALL.md [hunk 2]
```diff
@@ -275,6 +276,84 @@ librehealthehr/sites/default/letter_templates
...
+**Make sure that strict mode is disabled in Mysql**
+
+**[How to disable Mysql strict mode?](#how-to-disable-mysql-strict-mode-?)**
+
+#### Step 1
+Navigate to LibreEHR Setup:
+localhost/libreEHR/setup.php
+
+**Note:**
+1. Make sure that your XAMPP control panel has apache, mySQL, and Filezilla turned on
```
Comment (teryhill) [inline INSTALL.md]: Filezilla not needed

File: INSTALL.md [hunk 3]
```diff
@@ -275,6 +276,84 @@ librehealthehr/sites/default/letter_templates
...
+2. Apache needs to be on port 80, 443
+3. mySQL needs to be on port 3306
+4. Filezilla needs to be on port 21, 14147
+
+#### Step 2
+Leave default as the "Site ID:" and press continue.
+
+![First Step](./Documentation/1_Installing/images/windows_installation/Step_1.png)
+
+Make sure that there are no undefined index errors, if so make sure that you changed the php.ini file, or have the correct version of XAMP.
```
Comment (teryhill) [inline INSTALL.md]: XAMPP

File: INSTALL.md [hunk 4]
```diff
@@ -275,6 +276,84 @@ librehealthehr/sites/default/letter_templates
...
+![Sixth Step](./Documentation/1_Installing/images/windows_installation/Step_6.png)
+
+![Seventh Step](./Documentation/1_Installing/images/windows_installation/Step_7.png)
+
+![Eigth Step](./Documentation/1_Installing/images/windows_installation/Step_8.png)
+
+![Ninth Step](./Documentation/1_Installing/images/windows_installation/Step_9.png)
+
+**Yay, you have successfull setup LibreEHR!**
+
```
Comment (teryhill) [inline INSTALL.md]: I agree with your banner on the screen shots but others might take offence. Can you mask it please?

**Comment by aethelwulffe** [comment]
Two approvals.  Merging...  Good work folks!

**Comment by aethelwulffe** [comment]
Just checking...be advised that the original images were used to build the innosetup .exe xampp installer.  They were cropped to a size for that purpose.  This is an added directory, so that is fine if the originals remain in place.  These could still use a little cropping.

**Comment by robbyoconnor** [comment]
This stuff is amazing :stuck_out_tongue:

**Pull request merged**